### ISSUE TITLE
Migrate content from test WordPress Private Cloud instance to production

### ISSUE THREAD

**Issue by ty2k** [BCDevOps/developer-experience#2544]
Our existing [Private Cloud WordPress dev/test site](https://platform-services-dev.apps.silver.devops.gov.bc.ca/) contains pages that need to be migrated to an empty production WordPress instance that we will receive from the GDX Engagement team.

Using the migration strategy established in #2406, move the contents of the dev/test site to the production site.

**Additional context**
This migration should take place ASAP after receiving the production site, so that new content doesn't get added to the new environment before the existing content is in place. Any migration is destructive (replacing the target site content with the source site content, not "merging"), and we want to avoid having to manually copy and re-add any new content.

**Definition of done**
- [x] Pages are migrated
- [x] Plugins are either migrated or re-installed (and working)
- [x] Styling and themes are migrated
- [x] Content editor users are able to access the production site
- [x] Links between Tech Docs and Private Cloud WordPress site work

**Comment by mitovskaol** [comment]
@ty2k FYI we have purchased the https://servmask.com/products/multisite-extension plugin and it's been shared with GDX team to have it installed in our WP instances.

**Comment by ty2k** [comment]
GDX has done an initial import of our `-dev` site data to the new multi-site production instance. The production site is currently available through its OpenShift URL, as certificates and DNS are still needed to set up `cloud.gov.bc.ca` (per Shawn Turple). When we are ready to have our editors move their workflow over to the production site, we will need to coordinate a temporary halt to edits in `-dev`, a quick export from `-dev` and import to `-prod`, and messaging to our content editors that they should only be making changes in `-prod` going forward. It might be prudent to remove everyone's editing access in `-dev` and `-test` at this time so no new changes accidentally go into the wrong environment.

**Comment by ty2k** [comment]
I've unblocked this issue and moved it to in-progress despite not having DNS routing in place for `cloud.gov.bc.ca`. This was done after a meeting with Shawn and Olena on July 20 where we decided to do the migration from `dev` to `prod` ASAP while letting our content editors know not to add content to the `dev` site. This essentially puts us into "read-only" mode for content editors, though I am able to log into the production site and make changes (I have an edited `/etc/hosts` file pointing to the production IP).

I have completed a migration of content from our current single-instance `dev` site (https://platform-services-dev.apps.silver.devops.gov.bc.ca) to both our multi-site `test` site (https://cloud-test.apps.silver.devops.gov.bc.ca/private-cloud/) and our multi-site `prod` site (`cloud.gov.bc.ca`) . I have confirmed pages, plugins, themes, and styles moved over as expected.

For the final task in this thread, we need DNS/routing in place for `cloud.gov.bc.ca` so that our content editor users are able to access the production site and make edits there. I expect this to be in place within the next week. Shawn has some manual user cleanup to do on Tuesday, and I will wait on this to give our users access if DNS/routing is in place.

**Comment by ty2k** [comment]
On July 26, the Platform Services team (me, Olena, Luke) met with developers from GDX (Shawn, Jeff), and the Public Cloud team (Julian, Max) to change the DNS `A` record for `cloud.gov.bc.ca` through Remedy. The change succeeded and the DNS records propagated after a few minutes, making https://cloud.gov.bc.ca available to the internet.

I've confirmed that users are able to log in with KeyCloak and that they are assigned the role of "Subscriber with Read Private" as expected. I was able to have Matt and Martha log in and be assigned this role. I was able to change their permission level to "Editor" to allow them to edit content.

I've used SiteInspector to remove broken links and spelling errors from the site.

The site is generating an XML sitemap and it is not blocked by `robots.txt` or `noindex` tags.
